<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UWidgetPrintPlatijky</class>
 <widget class="QWidget" name="UWidgetPrintPlatijky">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Друк платіжок</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_platijkaNumber">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Початковий номер платіжки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_beginPlatijkaNumber">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_platijkaDate">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Дата платіжки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_platijkaDate">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_pererahunokMonth">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Перерахунок за місяць</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_pererahunokZaMisyac">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="displayFormat">
           <string>MM.yyyy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_readSumyOplat">
          <property name="text">
           <string>Всановити суми оплат</string>
          </property>
          <property name="icon">
           <iconset resource="../uPictRc.qrc">
            <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/view_text-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/view_text-256.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_organizacii">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Друк</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Призначення платежу</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сума</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Одержувач</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ЄДРПОУ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Банк Одержувача</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>МФО</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Рахунок</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ReadOnly</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_buttons">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_selectAll">
          <property name="text">
           <string>Позначити усі</string>
          </property>
          <property name="icon">
           <iconset resource="../uPictRc.qrc">
            <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/month-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/month-256.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deselectAll">
          <property name="text">
           <string>Зняти позначення</string>
          </property>
          <property name="icon">
           <iconset resource="../uPictRc.qrc">
            <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/1day-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/1day-256.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addRow">
          <property name="text">
           <string>Додати</string>
          </property>
          <property name="icon">
           <iconset resource="../uPictRc.qrc">
            <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/edit_add-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/edit_add-256.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deleteRow">
          <property name="text">
           <string>Видалити</string>
          </property>
          <property name="icon">
           <iconset resource="../uPictRc.qrc">
            <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/edit_remove-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/edit_remove-256.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>34</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_createCurPlatijka">
          <property name="text">
           <string>Створити поточну платіжку</string>
          </property>
          <property name="icon">
           <iconset resource="../uPictRc.qrc">
            <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/mail_new-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/mail_new-256.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_createSelectedPlatijky">
          <property name="text">
           <string>Створити позначені</string>
          </property>
          <property name="icon">
           <iconset resource="../uPictRc.qrc">
            <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/editcopy-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/editcopy-256.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableWidget_organizacii</tabstop>
 </tabstops>
 <resources>
  <include location="../uPictRc.qrc"/>
 </resources>
 <connections/>
</ui>
